tests/ImageSharp.Benchmarks/Drawing/FillRectangle.cs

```diff
@@ -0,0 +1,71 @@
+﻿// <copyright file="Crop.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageSharp.Benchmarks
+{
+    using System.Drawing;
+    using System.Drawing.Drawing2D;
+
+    using BenchmarkDotNet.Attributes;
+    using CoreImage = ImageSharp.Image;
+    using CoreRectangle = ImageSharp.Rectangle;
+    using CoreColor = ImageSharp.Color;
+    using System.IO;
+    using System.Numerics;
+
+    public class FillRectangle
+    {
+        [Benchmark(Baseline = true, Description = "System.Drawing Fill Rectangle")]
+        public void FillRectangleSystemDrawing()
+        {
+            using (Bitmap destination = new Bitmap(800, 800))
+            {
+
+                using (Graphics graphics = Graphics.FromImage(destination))
+                {
+                    graphics.InterpolationMode = InterpolationMode.Default;
+                    graphics.SmoothingMode = SmoothingMode.AntiAlias;
+                    var pen = new Pen(Color.HotPink, 10);
+                    graphics.FillRectangle(Brushes.HotPink, new Rectangle(10, 10, 190, 140));
+                }
+
+                using (MemoryStream ms = new MemoryStream())
```

Comment:
Don't save the bitmap in the benchmarks. We end up benchmarking the encoder also. Return the rectangle as a result BMDN works better with returns.